tests/CMakeLists.txt

```diff
@@ -125,3 +129,23 @@ FOREACH(T ${TESTS_CXX})
              ./${T})
 ENDFOREACH()
 
+# Tests only work on Unix at the moment
+if (UNIX)
+    FOREACH (T ${TESTS_PERF})
+        add_executable(${T} ${SOURCE_UTILS} ${T}.c)
+        target_include_directories(
+                ${T} PUBLIC
+                ../deps-build/${PLATFORM}/cmocka/include
+        )
+        target_link_libraries(${T} ${TESTLIB_OPTS_C})
+        add_test(${T} ${T})
+        # Removed to suppress false alarm: "--run-libc-freeres=no"
+        add_test(valgrind_${T}
+                valgrind
```

Comment:
In general, running test with valgrind will hurt performance. If you are adding performance tests, why are you running with valgrind?

Reply:
Just to make sure that the performance tests don't have any memory leaks. I don't collect any test results with valgrind.